mpas_analysis/ocean/ohc_timeseries.py

```diff
@@ -181,7 +188,7 @@ def ohc_timeseries(config, streamMap=None, variableMap=None):
         xlabel = "Time [years]"
         ylabel = "[x$10^{22}$ J]"
 
-        if ref_casename_v0 != "None":
+        if ref_casename_v0 != 'None':
             figname = "{}/ohc_{}_{}_{}.png".format(plots_dir,
```

Comment:
It's best to pick one and stick with it.  I prefer single quotes.  If I recall, there's no best practice as long as you're consistent.  We can make this part of the PEP8 clean up later on.  I just figured you had made a bunch of changes from double to single quotes, so might as well do a complete job of it.